CAMEL-15919: map java.io.File to swagger response schema type 'file'

eidottermihi opened the pull request:
Fixes CAMEL-15919 by setting schema type to 'file' if responseMessage class is 'java.io.File'.

davsclaus commented:
Wonder if you would take a look at camel-openapi-java to see if it has same problem.

eidottermihi commented:
Yes, camel-openapi-java has the same problem.

Updated the PR with a fix for camel-openapi-java and also updated the JIRA issue (added component camel-openapi-java).
